pkg/server/server.go

```diff
@@ -224,6 +233,9 @@ func (h *handler) authenticateEndpoint(w http.ResponseWriter, req *http.Request)
 	} else if userMapping, exists := h.lowercaseUserMap[arnLower]; exists {
 		username = userMapping.Username
 		groups = userMapping.Groups
+	} else if _, exists := h.accountMap[identity.AccountID]; exists {
```

Comment:
What do we want to do for the use case where the AccountID is mapped and the AWS User/Group is mapped to a list of groups but no username?  Should the username be required if there is a Role/User mapping?  Should we add an automapped username if it is not provided and automap accounts is present?

Reply:
I didn't want to handle that in this PR. As an explicit Role/User mapping exists, having a mapped AccountID doesn't really make a difference. Handling that case would mean we're changing the current behavior.